dev/devicelab/bin/tasks/ios_app_with_extensions_test.dart

```diff
@@ -28,50 +27,21 @@ Future<void> main() async {
 
       section('Create release build');
 
-      // This only builds the iOS app, not the companion watchOS app. The watchOS app
-      // has been removed as a build dependency and is not embedded in the app to avoid
-      // requiring the watchOS being available in CI.
-      // Instead, validate the tool detects that there is a watch companion, and omits
-      // the "-sdk iphoneos" option, which fails to build the watchOS app.
+      // This attempts to build the companion watchOS app. However, the watchOS
+      // SDK is not available in CI and therefore the build will fail.
+      // Check to make sure that the tool attempts to build the companion watchOS app.
       // See https://github.com/flutter/flutter/pull/94190.
       await inDirectory(projectDir, () async {
         final String buildOutput = await evalFlutter(
           'build',
           options: <String>['ios', '--no-codesign', '--release', '--verbose'],
         );
-        if (!buildOutput.contains('Watch companion app found')) {
-          throw TaskResult.failure('Did not detect watch companion');
-        }
-        if (buildOutput.contains('-sdk iphoneos -destination')) {
-          throw TaskResult.failure('-sdk must be omitted for app with watch companion');
+        if (!buildOutput.contains('-destination generic/platform=watchOS')) {
+          print(buildOutput);
+          throw TaskResult.failure('Did not try to get watch build settings');
         }
       });
 
-      final String appBundle = Directory(path.join(
-        projectDir.path,
-        'build',
-        'ios',
-        'iphoneos',
-        'Runner.app',
-      )).path;
-
-      final String appFrameworkPath = path.join(
-        appBundle,
-        'Frameworks',
-        'App.framework',
-        'App',
-      );
-      final String flutterFrameworkPath = path.join(
-        appBundle,
-        'Frameworks',
-        'Flutter.framework',
-        'Flutter',
-      );
-
-      checkDirectoryExists(appBundle);
-      await _checkFlutterFrameworkArchs(appFrameworkPath);
-      await _checkFlutterFrameworkArchs(flutterFrameworkPath);
-
       section('Clean build');
```

Comment:
Nit: this `clean` was pretty useless even before since the directory is removed right after this.  Can you remove it as well?

Reply:
Removed in https://github.com/flutter/flutter/pull/114078/commits/da87b9eadf57dcacd0609165e52ab450bafcbdc6.